### ISSUE TITLE
conda package incompatible with pytorch 1.11.0

### ISSUE THREAD

**Issue by ebetica** [conda-forge/fairscale-feedstock#12]
### Solution to issue cannot be found in the documentation.

- [X] I checked the documentation.

### Issue

```
conda install -d -c pytorch conda-forge::fairscale=0.4.6 pytorch::pytorch=1.11.0
...
  - package fairscale-0.4.6-cpu_py38h66f0ec1_0 requires pytorch >=1.10.2,<1.11.0a0, but none of the providers can be installed
```

I think fairscale should allow for the recently 1.11 release (no longer a0), this may be as simple as changing the version requirements...

### Installed packages

```shell
-
```

### Environment info

```shell
-
```

**Comment by h-vetinari** [comment]
The issues is that pytorch 1.11 hasn't been built for conda-forge yet, but hopefully it's close: https://github.com/conda-forge/pytorch-cpu-feedstock/pull/95

**Comment by hmaarrfk** [comment]
This is infact by design. Thank you for pointing it out @h-vetinari

**Comment by h-vetinari** [comment]
Builds for pytorch 1.11 have been merged through #14, should be installable in a few hours. 🙃
